angular/src/app/index/index.component.html

```diff
@@ -0,0 +1,39 @@
+<header>
+  <img src="./assets/images/header.png" alt="Oupsie, this doesn't work" width="100%" height="100%">
+</header>
+
+<div id="bg">
+  <div class="row valign-wrapper">
+    <div class="col l3 m4 center-align">
+      <img class="responsive-img hide-on-small-only" src="./assets/images/pipidon1.png" alt="Pipidon.png">
```

Comment:
This image's `alt` text should be empty as it is an [image used for ambiance](https://www.w3.org/WAI/tutorials/images/decorative/#image-used-for-ambiance-eye-candy).